### DISCUSSION TITLE
How to install the rewrite branch on OBS 28?

### DISCUSSION THREAD

**Discussion by RAMChYLD** [DistroAV/DistroAV#762] [Q&A - Support] [answered]
As per the title. 

I compiled obs-ndi rewrite and the compilation went trouble-free. However when it came to installing the plugin, it seems that OBS is extremely oblivious to it no matter what. 

I've tried everything including:
- Installing obs-ndi to $HOME/.config/obs-studio/plugins/obs-ndi/bin/64bit (didn't know this was a thing, but OBS didn't see it)
- Installing obs-ndi to /usr/lib/obs-studio/plugins/ (this was the old location for 4.9.1 and worked before the upgrade to 28.0.0)
- Installing obs-ndi to /usr/lib/x86_64-linux-gnu/obs-plugins (apparently the current location of the plugins)

None of them seem to work. Logs indicate that no matter where I put the plugin, OBS seem to pretend it doesn't exist.

Please help. I was supposed to start streaming an hour and a half ago and I'm losing my mind.

**Comment by RAMChYLD** [comment, marked as answer]
Edit: Made progress. After putting the plugin into /usr/lib/x86_64-linux-gnu/obs-plugins, the plugin and restarting obs numerous times, I noticed that the plugin is now recognized. However I'm unable to get the plugin to show things properly...

Edit 2: Nevermind. Turns out that the correct folder for the locales should be /usr/share/obs-plugins/obs-ndi/locale . not /usr/share/obs-plugins/obs-ndi/data/locale .

**Comment by squidds** [comment]
Does anyone have any feedback for the NDI runtime with the Alpha OBS plugin? I just installed it on my 1st PC and I am able to capture and send video. I still have OBS 27 and obs-ndi on my second PC. It seems to be fine so far for game capture. But I was wondering if anyone has experienced issues, before I try to use it in a stream. Thank you.

**Comment by RAMChYLD** [comment]
Well, as a receiver on Ubuntu, the plugin seems to be working okay. However, as a sender on Arch, the plugin transmits with a green tint. I'm not sure what gives given that the previous version works great- I'm forced to upgrade because Arch just upgraded to OBS 28.0.2 and have spent the last 3 hours packaging the files only to find that the plugin isn't working okay. 

My setup uses Plasma-Wayland KDE.
